### PR TITLE
Mailing

### PR DESCRIPTION
**Pull request by eggwhat**

### CODE REVIEW

File: SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Application/SwiftParcel.Services.Orders.Application/Exceptions/SmtpException.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace SwiftParcel.Services.Orders.Application.Exceptions
```
Comment (SaintAngeLs) [inline SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Application/SwiftParcel.Services.Orders.Application/Exceptions/SmtpException.cs:1]: Maybe, there is too small info here in the exception arguments

**Review by SaintAngeLs — COMMENTED** [review]
I get this issue, and also there are some problems in the Program.cs

```bash

...


                       

Unhandled exception. System.TypeInitializationException: The type initializer for 'QuestPDF.Settings' threw an exception.
 ---> QuestPDF.Drawing.Exceptions.InitializationException: The QuestPDF library has encountered an issue while loading one of its dependencies. This type of error often occurs when the current runtime is missing necessary NuGet packages. 

Please ensure the following NuGet packages are added to your primary/startup project and has matching versions:
- SkiaSharp.NativeAssets.Linux.NoDependencies
- HarfBuzzSharp.NativeAssets.Linux

For a detailed understanding, please examine the inner exception and consult the official SkiaSharp library documentation.
 ---> System.DllNotFoundException: Unable to load shared library 'libSkiaSharp' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: liblibSkiaSharp: cannot open shared object file: No such file or directory
   at SkiaSharp.SkiaApi.sk_compatpaint_new()
   at SkiaSharp.SKPaint..ctor()
   at QuestPDF.Helpers.NativeDependencyCompatibilityChecker.CheckIfExceptionIsThrownWhenLoadingNativeDependencies()
   --- End of inner exception stack trace ---
   at QuestPDF.Helpers.NativeDependencyCompatibilityChecker.Test()
   at QuestPDF.Settings..cctor()
   --- End of inner exception stack trace ---
   at QuestPDF.Settings.set_License(Nullable`1 value)
   at SwiftParcel.Services.Orders.Infrastructure.Extensions.AddBrevo(IConveyBuilder builder) in /home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Extensions.cs:line 162
   at SwiftParcel.Services.Orders.Infrastructure.Extensions.AddInfrastructure(IConveyBuilder builder) in /home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Extensions.cs:line 71
   at SwiftParcel.Services.Orders.Api.Program.<>c.<Main>b__0_0(IServiceCollection services) in /home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api/Program.cs:line 25
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.<>c__DisplayClass10_0.<ConfigureServices>b__0(WebHostBuilderContext _, IServiceCollection services)
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.BuildCommonServices(AggregateException& hostingStartupErrors)
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
   at SwiftParcel.Services.Orders.Api.Program.Main(String[] args) in /home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api/Program.cs:line 24
   at SwiftParcel.Services.Orders.Api.Program.<Main>(String[] args)
                                                                             
 ```

**Comment by SaintAngeLs** [comment]
It seems now the issue https://github.com/SaintAngeLs/courier_app/pull/56#pullrequestreview-1774102413 is resolved.

File: SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Brevo/Pdf/Documents/InvoiceDocument.cs
Comment (SaintAngeLs) [inline SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Brevo/Pdf/Documents/InvoiceDocument.cs:1]: Nice Work!!!

**Review by SaintAngeLs — APPROVED** [review]

**Review by an2508374 — APPROVED** [review]

File: SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Application/SwiftParcel.Services.Orders.Application/Exceptions/SmtpException.cs (hunk 1 above)
Comment (eggwhat) [reply SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Application/SwiftParcel.Services.Orders.Application/Exceptions/SmtpException.cs:1]: I don't know if much can be done here but I will try to do something about this when i will add mailing for two other cases.

**Pull request merged**